### ISSUE TITLE
Conflicting Dependency issues in requirement.txt

### ISSUE THREAD

**Issue by Sahaj-Srivastava24** [GDSC-IIIT-Kalyani/WOC-certificate-generator#33]
requirements.py has multiple versions of the same dependencies **charset-normalizer==2.0.4** and **charset-normalizer==2.0.6** which cause dependency conflicts while installation.

**Comment by rohitghughtyal** [comment]
yep one can be commented. same happens with <code> urllib3==1.26.6</code> and <code> urllib3==1.26.7</code> . I work on it to set it up entirely on my machine.
